specification/hybridcompute/resource-manager/readme.md

```diff
@@ -26,9 +26,19 @@ These are the global settings for the HybridCompute API.
 
 ``` yaml
 openapi-type: arm
-tag: package-preview-2021-01
+tag: package-preview-2021-03
 ```
 
+
+### Tag: package-preview-2021-03
```

Comment:
Please create new tag for the new apiVersion in readme.azureresourceschema.md as well